script/buildAddon.sh

```diff
@@ -3,9 +3,8 @@
 commit=2eab0008556a7e46289a9907d03e9b2534376c56
 
 rDIR=$(pwd)
-tmpDir=tmp_pg
```

Comment:
for some reason, node-gyp was having a hard time and blowing up during docker installs... so that's why I switched to `/tmp`